lib/clearance/configuration.rb

```diff
@@ -90,6 +90,11 @@ class Configuration
     # @return [ActiveRecord::Base]
     attr_accessor :user_model
 
+    # The array of explictly allowed environments to use Clearance::BackDoor in.
+    # Defaults to ["test", "ci", "development"]
+    # @return [Array<String>]
+    attr_accessor :allowed_envs
```

Comment:
Can we include "backdoor" in this config name so that it's clear what the envs are about?

Reply:
See the new name set in my changes.